src/coreclr/vm/eventing/eventpipe/ep-rt-coreclr.h

```diff
@@ -15,7 +15,6 @@
 #include "clrversion.h"
 
 #undef EP_ARRAY_SIZE
```

Comment:
Can this undef be deleted?

Reply:
Yup, it was a leftover. Next batch of changes are currently in [`feature/minipal2` branch](https://github.com/am11/runtime/tree/feature/minipal2). I'll wait for more feedback and push them to this branch later.